README.rst

```diff
@@ -21,23 +23,98 @@ pythx
         :target: https://coveralls.io/github/dmuhs/pythx?branch=master
 
 
+PythX is a library for the Mythx_ smart contract security analysis platform.
+
+.. code-block::
+
+        Usage: pythx [OPTIONS] COMMAND [ARGS]...
+
+        Options:
+        --help  Show this message and exit.
+
+        Commands:
+        check    Submit a new analysis job based on source code, byte code, or...
+        login    Login to your MythX account
+        logout   Log out of your MythX account
+        openapi  Get the OpenAPI spec in HTML or YAML format
+        ps       Get a greppable overview of submitted analyses
+        refresh  Refresh your MythX API token
+        report   Check the detected issues of a finished analysis job
+        status   Get the status of an analysis by its UUID
+        top      Display the most recent analysis jobs and their status
+        version  Print version information of PythX and the API
+
+
+By default, PythX comes enabled with a trial user:
```

Comment:
As the reader, it's not super clear to me whether the below codeblock is instructing me on how to login with my own credentials or that I need to explicitly log in as the trial user begin using PythX.

If it's the former 
```rst
By default, PythX comes re-enabled with a trial user. To login with your MythX account:
.. code-block::
        $ pythx login
        Please enter your Ethereum address [0x0000000000000000000000000000000000000000]:
        Please enter your MythX password [MySuperSecurePassword]:
        Successfully logged in as 0x0000000000000000000000000000000000000000
If you don't yet have an account, please visit https://mythx.io to register.
```

Reply:
Clarified that in 009cc573dc1125ec15ce645a5e48518abeabad38